Components/Image/image.css

```diff
@@ -0,0 +1,30 @@
+:root {
+    --black: #000000;
+    --medium-font-size: medium;
+    --large-font-size: large;
+    --larger-font-size: larger;
+    --bolder-font-weight: bolder;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+span {
+    font-weight: var(--bolder-font-weight);
+}
```

Comment:
you should give class name/ id while using span. don't style span directly

Reply:
Done 👍🏻